rest/config_legacy.go

```diff
@@ -4,9 +4,12 @@ import (
 	"flag"
 	"fmt"
 	"io"
+	"net/url"
 	"strings"
 	"time"
 
+	"gopkg.in/couchbaselabs/gocbconnstr.v1"
```

Comment:
Both libraries are (currently) the same, but `github.com/couchbase/gocbcore/v10/connstr` is the newer version/path for gocbv2, which I think we should be using instead.

```suggestion
	"github.com/couchbase/gocbcore/v10/connstr"
```